### PR TITLE
[Merged by Bors] - Lmm/update fluvio

### LINKED ISSUE #119: Update connectors to use latest stable version of fluvio
Also bump the versions of the connectors themselves so that docker publish picks them up.

### PR DESCRIPTION
**Pull request by morenol**
Closes #119

### CODE REVIEW

File: rust-connectors/sources/mqtt/src/main.rs [hunk 1]
```diff
@@ -81,12 +81,13 @@ fn main() -> Result<(), MqttConnectorError> {
 
         let mut mqttoptions = MqttOptions::new("rumqtt-async", mqtt_url, 1883);
         mqttoptions.set_keep_alive(timeout);
+        let producer = opts.common.create_producer().await?;
```
Comment (morenol) [inline rust-connectors/sources/mqtt/src/main.rs:84]: hopefully this will reduce the number of connections in the mqtt connector

**Review by simlay — APPROVED** [review]
lgtm

**Comment by simlay** [comment]
Huh. Interesting. You didn't have to update LLD. I've got that in my #116.

**Pull request closed without merging**